app.yaml

```diff
@@ -0,0 +1,12 @@
+name: "app"
+only-arches: [ i386 ]
+buildsystem: simple
+build-commands:
+  - "mkdir -p /app/share/DefleMask"
+  - "cp -R * /app/share/DefleMask"
+sources:
+  - type: archive
+    url: http://deflemask.com/DefleMask_Linux.tar.gz
```

Comment:
Does their license allow Flathub to redistribute it? If not, extra-data should be used instead of archive type.

Reply:
The upstream dev allows re-hosting of the files so i consider this solved

Reply:
You want to use `x-data-checker` to detect when this URL breaks due to an upgrade. See https://github.com/flathub/flatpak-external-data-checker for details.

Reply:
> You want to use `x-data-checker`

I'm unsure what to track on their official page and it looks like the download link is version-agnostic.

Reply:
You can add `'x-checker-data': true` to `flathub.json` – external-data-checker will detect that checksum is invalid and will submit a PR, which will also mean there was an upgrade. Not perfect, but better than nothing.